blog/templates/blog/post/detail.html

```diff
@@ -8,4 +8,25 @@ <h1>{{ post.title }}</h1>
         Published {{ post.publish }} by {{ post.author }}
     </p>
     {{ post.body|linebreaks }}
+
+    <!-- Comments section -->
+    <h2>Comments</h2>
+    {% for comment in comments %}
+        <div>
+            <p>{{ comment.text }}</p>
+            <p>By: {{ comment.author.username }}</p>
+            <p>Posted on: {{ comment.created }}</p>
+        </div>
+    {% empty %}
+        <p>No comments yet.</p>
+    {% endfor %}
+
+    <!-- Comment form -->
+    <h3>Add a comment</h3>
```

Comment:
This is already defined in the comment_form.html.

Reply:
I noticed sorry for that inconvenience i fixed it right away